baseband/helpers/sequentialfile.py

```diff
@@ -3,12 +3,92 @@
                         unicode_literals)
 
 import io
+import os
+import re
 import itertools
 from bisect import bisect
 import numpy as np
 from astropy.utils import lazyproperty
 
-__all__ = ['SequentialFileReader', 'SequentialFileWriter', 'open']
+__all__ = ['FileNameSequencer', 'SequentialFileReader', 'SequentialFileWriter',
+           'open']
+
+
+class FileNameSequencer(object):
+    """List-like generator of filenames using a template.
+
+    The template is formatted, filling in any items in curly brackets with
+    values from the header.  It is additionally possible to insert a file
+    number equal to the indexing value, indicated with '{file_nr}', and the
+    header time in ISO 8601 format, excluding fractions of a second, indicated
```

Comment:
I think we should not complicate the sequencer with `header_time` unless we actually use it in the code. Much better for people to be explicit about what they want.

Reply:
Agreed, and now removed.